usaspending_api/api_docs/markdown/documentation_index.md

```diff
@@ -8,7 +8,9 @@
 # Documentation Index <a name="documentation-index"></a>
 
 ## Tutorials <a name="tutorials"></a>
-* [USASpending API Introductory Tutorial](/docs/intro-tutorial) - A short tutorial covering basic API usage, generally covering: endpoints, filtering, and autocompletes.
+- [How to USe USAspending API Endpoints](https://www.youtube.com/watch?v=AEKL2LOkRZY) - A 10-minute video discussing how to use the USAspending API with couple of examples.
+  O [Side deck](https://www.usaspending.gov/data/Basic-API-Training.pdf) - with more detailed information about how to use USAspending API endpoints.
```

Comment:
I believe this `O` should be a `-` to properly bullet this list

Reply:
Also, I might change `with more` to just `More`. 

`Side Deck` should be `Slide Deck`

Reply:
done